ground/src/main/java/com/google/android/ground/ui/map/CameraUpdate.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.ui.map
+
+import com.google.android.ground.model.geometry.Point
+import com.google.android.ground.ui.home.mapcontainer.MapContainerViewModel
+
+data class CameraUpdate(
```

Comment:
Maybe we merge this with `CameraPosition`?

I only suggest this since we're not really making observable updates here. We construct some data we need to perform an observable update, but the actual updating part (applying the new camera location) is handled elsewhere.

We already have `CameraPositon` and some of the data are shared. I find it slightly difficult to understand the difference between these two classes at a glance, and it would simplify the number of classes we have.

wdyt?

Reply:
either way, feel free to do in another PR

Reply:
Thanks for the suggestion! I'll do it in a separate PR.